pom.xml

```diff
@@ -726,10 +726,52 @@
             </execution>
           </executions>
         </plugin>
-
+        <plugin>
+          <groupId>com.diffplug.spotless</groupId>
+          <artifactId>spotless-maven-plugin</artifactId>
+          <version>2.44.4</version>
+          <configuration>
+            <java>
+              <removeUnusedImports />
+              <eclipse>
+                <file>${project.basedir}/config/eclipse-format.xml</file>
```

Comment:
this provides the generic config for all IDE´s @romani denied the `ij_` related config keys. This is the `one to rule them all` solution used by maven. Developing there is a ease, as code fixes all checkstyle issues itself.